mlrun/runtimes/nuclio/application/application.py

```diff
@@ -344,24 +335,30 @@ def deploy(
             builder_env=builder_env,
         )
         logger.info(
-            "Successfully deployed function, creating API gateway",
-            api_gateway_name=self.status.api_gateway_name,
-            authentication_mode=authentication_mode,
+            "Successfully deployed function.",
```

Comment:
2 minor notes about log consistency that I noticed:
1. It's not very consistent across MLRun but we usually put the "successfully" at the end (e.g. `"Pipeline submitted successfully"`)
2. Either always end with a period or don't.

Just something to think about in general

Reply:
Dot at the end is subjected to having log kwargs ot not because the a `:` is appended. This should probably move to the logger (to add "." when needed)